src/wlapplication_messages.cc

```diff
@@ -113,6 +113,11 @@ void show_usage(const std::string& build_id, const std::string& build_type) {
 	               "                      depending on FPS. Useful in conjunction with\n"
 	               "                      --ai_training.")
 	          << endl
+	          << _(" --new_from_template=FILENAME\n"
+	               "                      Directly create a new singleplayer game\n"
+	               "                      from the given template file.")
```

Comment:
Suggestion:

"Directly create a new singleplayer game\n"
configured in the given file.  An example\
can be found in data/templates/new_game_template.")

Reply:
done